spec/services/redis_rate_limiter_spec.rb

```diff
@@ -4,9 +4,9 @@
   let(:now) { Time.zone.now }
 
   around do |ex|
-    REDIS_THROTTLE_POOL.with { |namespaced| namespaced.redis.flushdb }
+    REDIS_THROTTLE_POOL.with { |client| client.flushdb }
```

Comment:
Do we need to add any prefixes to RedisRateLimiter?

Reply:
Like here https://github.com/18F/identity-idp/blob/9880291efa7ba85d541a6a6a18e7bfac9cd08a25/app/services/redis_rate_limiter.rb#L50

Reply:
oops, yep, thank you, missed that one. fixed.